unocss/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "private": true,
+  "sideEffects": false,
+  "scripts": {
+    "build": "npm run build:css && remix build",
+    "build:css": "unocss './app/**/*.{js,ts,jsx,tsx}' -o ./app/styles/uno.css",
+    "dev": "concurrently \"npm run dev:css\" \"remix dev\"",
```

Comment:
Can we please use `npm-run-all` instead, like we do in our other examples & our templates?

https://github.com/remix-run/examples/blob/c149358fe3b1f7fd465c763209488ce67d0a6969/tailwindcss/package.json#L4-L13

Reply:
Sorry, didn't notice that. Will push a fix soon.